docs/configuration.rst

```diff
@@ -77,6 +77,9 @@ These settings affect the policy in the header. The defaults are in
     Set the ``report-uri`` directive. A tuple or list. Each URI can be a
     full or relative URI. *None*
     Note: This doesn't use default-src as a fall-back.
+``CSP_REPORT_TO``
+    Set the ``report-to`` directive. A dict. *None*
```

Comment:
We can update this from `A dict.` to something like `A string referring to a reporting group from a report-to header served separately` and link to the draft standard.

Reply:
Updated. I've made mention of the examples in section 1.2 here
https://w3c.github.io/reporting/#group

Reply:
If all is well I can squash the commits down before doing a merge too.